src/lib/artworkMediums.ts

```diff
@@ -4,12 +4,14 @@ export default {
     long_description:
       "Includes architectural models; buildings (e.g., house, temple).",
     name: "Architecture",
+    mediumFilterGeneSlug: "architecture-1",
```

Comment:
Strange but true. There are two genes that were called "Architecture" at one point or another: one for artistic discipline, and one for subject matter. We're interested in the first one, but it was created later and ended up with this janky slug.